custom_components/frigate/sensor.py

```diff
@@ -325,6 +339,14 @@ def device_info(self) -> DeviceInfo:
             "via_device": get_frigate_device_identifier(self._config_entry),
             "name": get_friendly_name(self._cam_name),
             "model": self._get_model(),
+            "configuration_url": self._config_entry.data.get(
+                CONF_URL, self._config_entry.data.get(CONF_HOST, "")
```

Comment:
Ditto.

Reply:
And these to:
```python
"configuration_url": f"{self._config_entry.data[CONF_URL]}{'/cameras/' + self._cam_name if self._cam_name not in get_zones(self._frigate_config) else ''}"
```
I don't know that that's any more readable 😅

Reply:
I think the one in your comment is more readable, it can also be condensed further: `/cameras/{self._cam_name}`

Reply:
You know, I hadn't considered that pointing to `/cameras/` alone wouldn't cause an issue. That would be fine, so I'll implement these in a bit.